frontend/src/pages/PageTimings.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
It would be better to move page under directory SpeedOverview, or something like that, in order to reduce trash inside directory